cmd/otelcol/Dockerfile.windows

```
@@ -27,6 +27,22 @@ RUN Add-Type -AssemblyName System.IO.Compression.FileSystem ;\
     [System.IO.Compression.ZipFile]::ExtractToDirectory('agent-bundle_windows_amd64.zip', '.')
 RUN Remove-Item agent-bundle_windows_amd64.zip -force
 
+# Download JMX Metric Gatherer
+ARG JMX_METRIC_GATHERER_RELEASE
+RUN New-Item -Path "C:\\" -Name "opt" -ItemType "directory"
+RUN Invoke-WebRequest -Uri "https://github.com/open-telemetry/opentelemetry-java-contrib/releases/download/v${env:JMX_METRIC_GATHERER_RELEASE}/opentelemetry-jmx-metrics.jar" -Outfile "c:\opt\opentelemetry-java-contrib-jmx-metrics.jar"
+
+ARG JDK_VERSION="11.0.19_7"
+ARG OPENJDK_BASE_URL="https://github.com/adoptium/temurin11-binaries/releases/download"
+ARG JAVA_HOME="C:\java\openjdk-${JDK_VERSION}\bin"
+RUN Set-Variable -Name "ENCODED_VER" -Value "\"$env:JDK_VERSION\".replace(\"_\", \"%2B\")"; \
+    Invoke-WebRequest -Uri "${env:OPENJDK_BASE_URL}/jdk-${ENCODED_VER}/OpenJDK11U-jdk_x64_windows_hotspot_${env:JDK_VERSION}.zip" -OutFile "openjdk.zip" ; \
+    Expand-Archive "openjdk.zip" -DestinationPath "C:\java" ;
+RUN setx PATH "$env:JAVA_HOME;$env:PATH"
```

Not sure if this will set the env var correctly.  Maybe something like the following will work better:
```suggestion
RUN powershell.exe -Command $path = "${env:JAVA_HOME}\bin" + $env:path; Set-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\Session Manager\Environment\' -Name Path -Value $path
```
See https://github.com/MicrosoftDocs/Virtualization-Documentation/blob/main/windows-container-samples/golang/Dockerfile#L15.

---

It looks like we're using the servercore images, so I ended up following the [syntax there instead.](https://github.com/MicrosoftDocs/Virtualization-Documentation/blob/main/windows-container-samples/golang/Dockerfile.WindowsServerCore)